tests/collecting/CollectorsTest.py

```diff
@@ -30,7 +30,10 @@ def test_file_invalid(self):
         self.assertEqual(collect_files(['invalid_path'],
                                        self.log_printer), [])
         self.assertEqual([log.message for log in self.log_printer.logs],
-                         ["No files matching 'invalid_path' were found."])
+                         ["No files matching 'invalid_path' were found. "
```

Comment:
I see. 
I didn't even think about that. 
However, this would not be consistent to  [Line 177 of current Collectors.py](https://github.com/coala/coala/blob/master/coalib/collecting/Collectors.py).

So should I include escaped single quotes for both occasions, or leave it as it is?